README.md

```diff
@@ -1,2 +1,176 @@
-# capstone_prize-giving_day
-This is prize giving web page.
+# TriumphTribute
+Assessment-project
+<a name="readme-top"></a>
+<div align="center">
+  
+  <h3><b>TriumphTribute</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [TriumphTribute] <a name="Assessment project for the portal module"></a>
+
+>This is an assessment project for all of the frontend lessons in the first module.
+
+**[TriumphTribute]**  .
+
+## 🛠 Built With <a name="HTML/CSS"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+This project is build using html and css and javascript.
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Header Section]** 
+Contains a mobile humburger menu inthe mobile version
+- **[parners section]**
+Contains partners logos.
+- **[footer]**
+COntains the meeting logo and the copyright
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Live Demo  - click [TriumphTribute]()
```

Comment:
[ ] kindly add the link to your deployed website here, so your readme looks more professional.🤓